NAMESPACE

```diff
@@ -110,10 +109,10 @@ export(runSensitivity)
 export(saveTaskResults)
 export(setDefaultBins)
 export(setDefaultNumericFormat)
+export(setDefaultPlotFormat)
 export(setDefaultStairstep)
 export(setDefaultTheme)
 export(setDefaultThemeFromJson)
-export(setPlotFormat)

```

Comment:
@pchelle We should try to keep the changes of the **public** RE interface as small as possible.
If there are good reasons for refactoring - sure, we should make changes. 
But here I would suggest to keep the naming `setPlotFormat` even if it's not optimal. Or: 
* export both functions (`setPlotFormat `and `setDefaultPlotFormat`)
* `setPlotFormat `should just call `setDefaultPlotFormat`